From d53e3a14504fa5f4608e43b4aa3475754c3759d2 Mon Sep 17 00:00:00 2001
From: d k <de_005@yahoo.com>
Date: Fri, 15 Mar 2013 05:52:23 -0700
Subject: [PATCH] init

---
 test.htm | 5 +++++
 1 file changed, 5 insertions(+)
 create mode 100644 test.htm

diff --git a/test.htm b/test.htm
new file mode 100644
index 0000000..46f12e0
--- /dev/null
+++ b/test.htm
@@ -0,0 +1,5 @@
+<body>
+
+hi world
+
+</body>
\ No newline at end of file
-- 
1.7.11.1

